src/main/java/de/tum/in/www1/artemis/repository/TextBlockRepository.java

```diff
@@ -28,4 +28,6 @@
 
     @EntityGraph(type = LOAD, attributePaths = { "submission" })
     Set<TextBlock> findAllBySubmissionIdIn(Set<Long> submissionIdList);
+
+    void deleteAllBySubmission_Id(Long submissionId);
```

Comment:
I think the underscore is not necessary because it wouldn't be consistent with the other method names: `deleteAllBySubmissionId`

Reply:
I tried this and it does not work, so I'll keep the `_` (which is part of some black magic of Hibernate).